### ISSUE TITLE
Pypugjs error caused by missing force_text in Django 4.0

### ISSUE THREAD

**Issue by wouterha** [kakulukia/pypugjs#72]
* pypugjs version: 5.9.9
* Django version: 4.0
* Python version: 3.8
* Operating System: linux

### Description

Pypugjs does not load in Django 4.0 because django.utils.encoding.force_text doesn't exist anymore.
It seems that force_text can be replaced by force_str according to this issue
https://stackoverflow.com/questions/70382084/import-error-force-text-from-django-utils-encoding

### What I Did
Added pypugjs to django settings.py as follows:
```
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [os.path.join(BASE_DIR, 'templates')],
        'OPTIONS': {
            'context_processors': [
                'django.template.context_processors.debug',
                'django.template.context_processors.request',
                'django.contrib.auth.context_processors.auth',
                'django.contrib.messages.context_processors.messages',
            ],
            # PyPugJS support:
            'loaders': [
                ('pypugjs.ext.django.Loader', (
                    'django.template.loaders.filesystem.Loader',
                    'django.template.loaders.app_directories.Loader',
                ))
            ],
            'builtins': [
                'pypugjs.ext.django.templatetags',
            ],
        },
    },
]
```

Relevant part of error:
```
File "/lib/python3.8/site-packages/pypugjs/ext/django/compiler.py", line 2, in <module>
  from django.utils.encoding import force_text as to_text
ImportError: cannot import name 'force_text' from 'django.utils.encoding' (/lib/python3.8/site-packages/django/utils/encoding.py)

```

**Comment by kakulukia** [comment]
I will look into this soon, if you could provide a PR this would help a lot.

**Comment by kakulukia** [comment]
This should be solved with 5.9.10 .. unfortunately travis is not working right now, but i tested at least some versions manually and it should be ok.
